inyoka_theme_default/templates/confirm_action_flash.html

```diff
@@ -0,0 +1,17 @@
+{#
+  confirm_action_flash.html
+  ~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  :copyright: (c) 2011-2015 by the Inyoka Team, see AUTHORS for more details.
```

Comment:
Please change all the copyright-dates to `2013-2015`. (→ does not only affect this file)